src/instanceMethods/close.js

```diff
@@ -99,6 +87,32 @@ export function close (resolveValue) {
   delete this.params
 }
 
+const handlePopupAnimation = ({ popup, innerParams : { onAfterClose } }) => {
+  const container = dom.getContainer()
+  // If animation is supported, animate
+  const animationIsSupported = dom.animationEndEvent && dom.hasCssAnimation(popup);
+
+  if (animationIsSupported) {
+    animatePopup({ popup, container, onAfterClose })
```

Comment:
what's the deal with passing one object instead of multiple params into aa function?

Reply:
I relied on this [repository](https://github.com/ryanmcdermott/clean-code-javascript#functions), with some clean code tips. Where I currently work, restructuring in this way has helped to give more readability in calling functions

Reply:
Sorry for the delayed reply @JulianoGTZ and thanks for linking the `clean-code-javascript` repo, it's very useful!

While I agree on the example in the repo, I disagree that using that technique improves readability in this PR. Let me explain on examples:

---

The example from the `clean-code-javascript` repo:

```js
// with object
function createMenu({ title, body, buttonText, cancellable }) { ... }
createMenu({
  title: "Foo",
  body: "Bar",
  buttonText: "Baz",
  cancellable: true
});

// without object
function createMenu(title, body, buttonText, cancellable) { ... }
createMenu({
  "Foo",
  "Bar",
  "Baz",
  true
});
```

Clearly, using the object is way better in terms of readability.

---

In our case:

```js
// with object
const animatePopup = ({ popup, container, onAfterClose }) => { ... }
animatePopup({ popup, container, onAfterClose })

// without object
const animatePopup = (popup, container, onAfterClose) => { ... }
animatePopup(popup, container, onAfterClose)
```

Almost no difference in terms of readability, in fact the first one (with object) is a bit harder to read.

So, if you're okay, let's not introduce objects in this PR.

Reply:
hi @limonte. I agree totally, especially with the example.

Apart from contributing to the community and with this repository, I expected to obtain this type of learning.

Really, thank you very much.